### DISCUSSION TITLE
TFT Display power save

### DISCUSSION THREAD

**Discussion by vale-max** [rern/rAudio#333] [General]
Hi @rern ,
is there a way to set a power saving timeout (or a blanking screensaver) for the TFT display?

**Comment by ceejayemm** [comment]
Under Features / Others / Browser on RPi click on the cog wheel where there is an option to set a 'Screen Off' timer in minutes.

Chris

**Comment by vale-max** [comment]
Ok, so I tried the setting and it does blank the screen but the backlight does not turn off, so I did some investigation and found that the screen I bought on [aliexpress](https://it.aliexpress.com/item/32918104361.html) differently from the screen in the article description has a LED_FUNCTION jumper, which comes hardwired to OFF.

I researched a bit and found this PDF on [mouser](https://www.mouser.com/pdfDocs/SparkFun-lcd-17520-touchscreen.pdf) which describes a TFT screen with some "LED_FUNCTION" wired to pin 15 (BCM_22).

So I took my soldering iron, removed the 1k SMD resistor shunting the OFF pads (making it fly and losing it), and soldered a 1k resistor on the ON pads, like in the pic below.

Now the backlight can be satisfactorily shut off or turned on with command "gpio write 3 0/1"

Further researches and found how to have a working /sys/class/backlight node, in this [post](https://www.raspberrypi.org/forums/viewtopic.php?t=201639#p1255688) (correct GPIO value is 22)

The problem now is that dpms off does not turn off the backlight anyways :disappointed: 

Until now the best candidate as solution seems [this](https://github.com/notro/fbtft-spindle/wiki/Appendix#x-windows-backlight-script)

Finally the image of the circuit side of my TFT screen with the huge resistor added by me.

![IMG_20210525_202309](https://user-images.githubusercontent.com/54400080/119577867-b9e10d00-bdbb-11eb-983c-372b8ac40b8b.jpg)
